Extensions/xkit_patches.js

```diff
@@ -260,6 +260,23 @@ XKit.extensions.xkit_patches = new Object({
 				});
 				return posts;
 			};
+
+			XKit.interface.post_window.state = () => {
+				let post_state = $("#post_state").val();
+
+				return {
+					publish: post_state == "0",
+					draft:   post_state == "1",
+					queue:   post_state == "2",
+					private: post_state == "private"
+				};
+			};
+
+			XKit.interface.post_window.blog =
+				() => $("#channel_id").val() || $("span.caption").text();
```

Comment:
this is too broad of a selector...... anything can have a caption, and there's no way for us to know which one is the post_window. I think a good selector here would be `$(".post-form--header [data-js-tumbleloglabel]").text()`

Reply:
OK (although there *seems* to only ever be one `span.caption`)